charts/piraeus/templates/operator-controller.yaml

```diff
@@ -45,6 +45,8 @@ metadata:
   namespace: {{ .Release.Namespace }}
   annotations:
     helm.sh/resource-policy: keep
+    helm.sh/hook: pre-install
+    helm.sh/hook-delete-policy: before-hook-creation
```

Comment:
up to consider

Reply:
Is this to prevent overriding the secret on updates?

Reply:
Yes, it will install secret only during `helm install`, then it will never override it during `helm upgrade` even if it was removed.